### DISCUSSION TITLE
Git use by Vemto

### DISCUSSION THREAD

**Discussion by ballesta** [TiagoSilvaPereira/vemto-issues#176] [Q&A] [answered]
Hi,

What are the Git usage in a Vemto App.

When should we use 
- the default Laravel Git repository
- our on private Git repository 

What is the use case of the two alternatives?

Best regards 
Bernard

**Comment by TiagoSilvaPereira** [comment, marked as answer]
Hi @ballesta,

Vemto uses the Laravel repository (https://github.com/laravel/laravel) to clone the project base (*similar to run **laravel new** from your command line*). You can use another directory if you want to clone a custom project base (but I only recommend it if you know exactly what you are doing, as it may break your project if using the incorrect base).

Cheers
